docs/docker_quickstart.md

```diff
@@ -67,29 +67,33 @@ Create a new directory and place the [docker-compose file](https://raw.githubuse
         # image: freqtradeorg/freqtrade:develop_pi
         ```
 
-=== "ARM64 (Mac M1)"
-    Make sure that your docker installation is running in native mode
+=== "ARM 64 Systenms (Jetson Nano, Mac M1, Raspberry Pi 4 8GB)"
+    In case of a Mac M1, make sure that your docker installation is running in native mode
 
     ``` bash
     mkdir ft_userdata
     cd ft_userdata/
-    # Download the docker-compose file from the repository
-    curl https://raw.githubusercontent.com/freqtrade/freqtrade/stable/docker-compose.yml -o docker-compose.yml
 
-    # Edit the compose file, uncomment the "build" step and use "./docker/Dockerfile.aarch64"
-    # Also, change the image name to something of your liking
+    # arm64 images are not yer provided via Docker Hub and need to be build locally first. Depending on the device,
+    # this may take a few minutes (Apple M1) or up to two hours (Raspberry Pi)
+    git clone  https://github.com/freqtrade/freqtrade.git
+    docker build -f ./freqtrade/docker/Dockerfile.aarch64 -t freqtradeorg/freqtrade:develop_arm64 freqtrade
```

Comment:
Don't recommend doing it this way - but instead use docker-compose to build the image.

The compose-file will need to be edited anyway (the image name is now different) - so using `docker-compose build` will be the simpler command (and will also be simpler in update-scenarios).

Reply:
Fair enough. Personally, I think that gets a little bit more tedious when you are trying to build anything on top of the other Dockerfiles inside `./docker`. But if you prefer it that way, I'll happily change that.

Reply:
usually, mixing docker + docker-compose is quite confusing to people not used to docker (at least that's my experience from trying to explain it to some colleagues at work) ...